src/MeshGroup.h

```diff
@@ -30,6 +30,15 @@ class MeshGroup : public NoCopy
     void clear();
 
     void finalize();
+
+	/*!
+	 * Scale the entire mesh group.
+	 *
+	 * The mesh group is scaled around the bottom center of its bounding box. So
+	 * that's the center in the X and Y directions, but Z=0. This simulates the
+	 * shrinkage while sticking to the build plate.
+	 */
+	void scale(const Ratio factor);
```

Comment:
I think it would be better to name it "scaleFromBottom" or something like that. Scale by a factor to me tends to mean that it's doing the scale from the center (the documentation does say it though, so it's not really a big remark).

Reply:
Agreed! Is done now.